app/models/value_sorter.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+class ValueSorter
+  class << self
+    def sort_by_localized_name(values, locale: "en")
+      sort_by_proc = proc { |value| value.name(locale: locale) }
+
+      perform_sort(values, sort_by_proc)
+    end
+
+    private
+
+    def perform_sort(values, sort_by_proc)
+      sorted_values = values.sort do |a, b|
+        value_a = sort_by_proc.call(a)
+        value_b = sort_by_proc.call(b)
+
```

Comment:
The use of a proc here feels weird/unnecessary?

Reply:
Particularly b/c `perform_sort` is private and only used by this one method

Reply:
It's how it's handled in the gem. It allows it to be more flexible later if we want to use it elsewhere. But if we're okay with making it specific to this use case for now I can just update the sort method to access name directly and skip the extra step with the proc